source/_docs/configuration/events.markdown

```diff
@@ -45,29 +33,155 @@ Event `call_service` is fired to call a service.
 | `service_data`    | Dictionary with the service call parameters. Example: `{ 'brightness': 120 }`. |
 | `service_call_id` | String with a unique call id. Example: `23123-4`.                              |
 
-## Event `automation_reloaded`
+### `component_loaded`
+
+This event is fired when a new integration has been loaded and initialized.
```

Comment:
Adding a little note to this one as a suggestion.

```suggestion
This event is fired when a new integration has been loaded and initialized.

Please note, that while this event is fired for each loaded integration during Home Assistant startup, the automation engine of Home Assistant is started last. Thus this event can not be used to automate on during startup as it would have missed these events.
```